src/libs/ReportUtils.ts

```diff
@@ -545,6 +546,13 @@ function isReportApproved(report: OnyxEntry<Report>): boolean {
     return report?.stateNum === CONST.REPORT.STATE_NUM.SUBMITTED && report?.statusNum === CONST.REPORT.STATUS.APPROVED;
 }
 
+/**
+ * Checks if the supplied report is an expense report in Open state and status.
+ */
+function isDraftExpenseReport(report: OnyxEntry<Report>): boolean {
```

Comment:
Just saw this method... why, oh, why did we introduce a new term `draft` for what in every other place we call `open`? 😢

Reply:
I followed the terminology used in the App so its clearer for external engineers who dont have context of the Open reports, however, I agree in the hindsight it would be better to follow the backend terminology and keep `Drafts` as the marketing word only 👍